### ISSUE TITLE
Switzerland page on radiocells.org

### ISSUE THREAD

**Issue by amilopowers** [wish7code/openbmap#113]
The Switzerland page on radiocells.org is somehow messed up.

http://radiocells.org/country/ch

There are only three cell providers in Switzerland: https://de.wikipedia.org/wiki/Schweizer_Mobilfunkmarkt
So only Swisscom, Sunrise, and Salt (was Orange) have their proper cellular network. Orange does not exist as a brand anymore it is now branded and named "Salt".
There is a GSM-R network from SBB though.

All those weird ones below (like Tele2) do not have a network.

I don't know if that's the right place to put that... I guess a wiki-style website would be better to adapt fast.

**Comment by wish7code** [comment]
Hey Ulrich,

thanks for the report! I'll update accordingly once we're done with the ongoing server relocation.

While not having a real wiki-like interface you could still report errors even today
(scroll down http://radiocells.org/cell_provider/796, press **report error**)
Agreed that's not the best from a UI perspective, but only a pragmatic solution :-)

**Comment by amilopowers** [comment]
Ok I didn't see that link at the bottom of the page.

How long does the server migration take?

**Comment by amilopowers** [comment]
I saw two more things:

The international calling code is "+41".

Salt has a new color so you might change it to neon green as well. www.salt.ch

**Comment by wish7code** [comment]
> How long does the server migration take?

Always longer than expected :D Seriously, after one week with very bad connection everything seems to normalize.. The server still needs some time to process all the uploads in the mean time..

I fixed the calling code accordingly. I also dumped MVNO as suggested by you. MVNO were only listed for the sake of completeness anyways, as I'm not aware of any MVNO reporting it's official  MCC/MNC call sign. Instead all MVNO I know report the actual cell operators MCC/MNC (e.g. Sunrise)
